exporter/exporterhelper/logs.go

```diff
@@ -114,6 +122,70 @@ func NewLogsExporter(
 	}, err
 }
 
+type LogsConverter interface {
+	// RequestFromLogs converts plog.Logs data into a request.
+	RequestFromLogs(context.Context, plog.Logs) (Request, error)
+}
+
+// NewLogsExporterV2 creates new logs exporter based on custom LogsConverter and RequestSender.
+func NewLogsExporterV2(
```

Comment:
nit: We could name this `NewLogsRequestExporter` instead of using the `V2` suffix

Reply:
We have not stabilized the previous API, so I agree with @mx-psi on this.

Reply:
Also would be good if we can eventually make old func call into the new.

Reply:
SGTM. Updated

Reply:
> Also would be good if we can eventually make old func call into the new.

I didn't add queue options for the new helper in this PR yet. Wanted to do it as the next step. Once it's done, we can make the old func call the new one.